<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_SelDefSeries</class>
 <widget class="QDialog" name="Dialog_SelDefSeries">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Define Series</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableView" name="tableView"/>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>序列起始值:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_Start">
           <property name="decimals">
            <number>10</number>
           </property>
           <property name="minimum">
            <double>-999999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>序列增值</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_Detal">
           <property name="decimals">
            <number>10</number>
           </property>
           <property name="minimum">
            <double>-999999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Cannel">
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_OK">
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_SelDefSeries</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Cannel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_SelDefSeries</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
